### PR TITLE
feature: try to lookup asset by Pimcore's own asset URL

### LINKED ISSUE #160: Deduplicate assets in asset URL interpreter by their Pimcore URL too
| Q                | A
| ---------------- | -----
| Bug report?      | no
| Feature request? | yes
| BC Break report? | no
| RFC?             | no
| Branch?          | master

If you at some point export your Pimcore dataset, you'll likely export the assets with their Pimcore URLs (to enable them being fetched from there by third parties).

But, if you now import that dataset back to Pimcore and use `Deduplicate by URL` (the URL it was originally imported from), will not match Pimcore's URL (since it might have been imported to Pimcore from elsewhere.

We could also check if we can find the asset via its Pimcore's URL too, not just target Pimcore path or original URL.

### PR DESCRIPTION
**Pull request by dkarlovi**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | #160 

Closes #160.

### CODE REVIEW

File: src/ImportDefinitionsBundle/Interpreter/AssetUrlInterpreter.php [hunk 1]
```diff
@@ -52,14 +53,15 @@ public function interpret(Concrete $object, $value, Mapping $map, $data, Definit
...
+                // URL seems to be pointing to our own asset URL, try to lookup by that
+                $assetFullPath = str_replace($assetsUrlPrefix, '', $value);
 
-            // Convert placeholder path
-            $context = new PlaceholderContext($data, $object);
-            $assetPath = $this->placeholderService->replace($path, $context);
-            $assetFullPath = sprintf('%s/%s', $assetPath, $filename);
-            
-            if ($configuration['deduplicate_by_url']) {
+                $asset = Asset::getByPath($assetFullPath);
```
Comment (dkarlovi) [inline src/ImportDefinitionsBundle/Interpreter/AssetUrlInterpreter.php]: This can be `null` so need to account for that.

Comment (dpfaffenbauer) [reply src/ImportDefinitionsBundle/Interpreter/AssetUrlInterpreter.php]: we should, otherwise it will throw an null-exception ;)

Comment (dkarlovi) [reply src/ImportDefinitionsBundle/Interpreter/AssetUrlInterpreter.php]: Indeed. :)

**Comment by dkarlovi** [comment]
@dpfaffenbauer this one is ready.

**Comment by dpfaffenbauer** [comment]
thanks a lot

**Pull request merged**